nio/client/base_client.py

```diff
@@ -985,6 +986,9 @@ def receive_response(
             self._handle_room_forget_response(response)
         elif isinstance(response, ToDeviceResponse):
             self._handle_olm_response(response)
+        elif isinstance(response, RoomGetEventResponse):
+            if isinstance(response.event, MegolmEvent):
+                response.event = self.decrypt_event(response.event)
```

Comment:
This can raise an exception if the event can't be decrypted, also it should be only called if `self.olm` isn't None.

Reply:
Done